src/Microsoft.PowerShell.Commands.Utility/commands/utility/Write-Object.cs

```diff
@@ -55,10 +55,8 @@ protected override void ProcessRecord()
             {
                 enumerate = false;
             }
-            foreach (PSObject inputObject in _inputObjects) // compensate for ValueFromRemainingArguments
-            {
-                WriteObject(inputObject, enumerate);
-            }
+
+            WriteObject(_inputObjects, enumerate);
```

Comment:
Hmm, that's tricky.  In fact I'd say it's been a bug in `Write-Output` all along.

```posh
Write-Output aa,bb cc,dd

# is equivalent to

Write-Output -InputObject @( @('aa', 'bb'), @('cc', 'dd') )
```

Either way, `InputObject` is a 2-element array, and each element also happens to be a 2-element array.  What I would expect to happen here is what happens after this PR:  You send two objects down the pipeline, both of which are 2-element arrays.  If you added the `-NoEnumerate` switch, then you would send one object down the pipeline, which is a 2-element array containing two other 2-element arrays.  (In fact, with the -NoEnumerate switch, you should only _ever_ send one object, an array, down the pipeline, which is sort of the point.)

Today, `Write-Output aa,bb cc,dd` instead sends 4 strings down the pipeline.  It enumerates two levels deep instead of one.  If you use the `-NoEnumerate` switch on that command today, you get two 2-element arrays going down the pipeline (what I'd expect to see without `-NoEnumerate`).

Without this change, several other `Write-Output` tests failed when I ran `Start-PSPester`.  With this change, everything was green, but apparently that just means that the test coverage isn't good enough right now.